galactic/distribution.yaml

```diff
@@ -2665,6 +2665,21 @@ repositories:
       url: https://github.com/open-rmf/rmf_internal_msgs.git
       version: galactic
     status: developed
+  rmf_task:
+    doc:
+      type: git
+      url: https://github.com/open-rmf/rmf_task.git
+      version: main
+    release:
+      tags:
+        release: release/galactic/{package}/{version}
+      url: https://github.com/ros2-gbp/rmf_task-release.git
+      version: 1.0.0-1
+    source:
+      type: git
+      url: https://github.com/open-rmf/rmf_task.git
+      version: main
```

Comment:
@marcoag just to double check, should this be the `galactic` branch instead? Same for the `doc` entry

Reply:
Thanks for the catch! I just updated it.